### PR TITLE
Add namespaces and remove unused data

### PR DESCRIPTION
**Pull request by abrca**
### Description of Change ###

Namespace is automatically added to a using directive at the beginning of the code if it doesn't already exist.
Remove unused Literals

### Issues Resolved ### 

None

### API Changes ###
 
None

### Platforms Affected ### 

- All

### Behavioral Changes ###

None

### PR Checklist ###

- [ ] I have included examples or tests
- [ ] I have updated the change log
- [ ] I am listed in the CONTRIBUTORS file
- [ ] Rebased on top of the target branch at time of PR
- [ ] Changes adhere to coding standard

### CODE REVIEW

File: snippets/C#/vmtaskcommand.snippet [hunk 1]
```diff
@@ -35,6 +35,11 @@ private async Task On$name$ExecuteAsync()
     // TODO: Handle command logic here
 }]]>
             </Code>
+            <Imports>
+                <Import>
+                    <Namespace>Catel.MVVM</Namespace>
```
Comment (GeertvanHorrik) [inline snippets/C#/vmtaskcommand.snippet:40]: Shouldn't this one also import System.Threading.Tasks?

File: snippets/C#/vmtaskcommandwithcanexecute.snippet [hunk 2]
```diff
@@ -40,6 +40,11 @@ private async Task On$name$ExecuteAsync()
     // TODO: Handle command logic here
 }]]>
             </Code>
+            <Imports>
+                <Import>
+                    <Namespace>Catel.MVVM</Namespace>
```
Comment (GeertvanHorrik) [inline snippets/C#/vmtaskcommandwithcanexecute.snippet:45]: Shouldn't this one also import System.Threading.Tasks?

**Review by GeertvanHorrik — CHANGES_REQUESTED** [review]
Fantastic PR, thanks for the great improvements. Please see the comments and double check whether they make sense.

**Comment by abrca** [comment]
You are right, I'll fix it.

And I have a question about model.snippet.
It use '#if NET', and this symbol is defined in \templates\C#\ProjectTemplates\Catel.WPF.Application\Catel.WPF.Application.csproj as synonym to .NET Framework target.
Keep it, assuming that people using this snippets will also use this project template, or replace it with more correct '#if NETFRAMEWORK || NETCOREAPP || NETSTANDARD', as in Catel.Core\Data\ModelBase.cs ?

**Comment by GeertvanHorrik** [comment]
I think the serialization constructor (at least for 6.0) can be removed. And since we are moving towards a single .NET anyway, I think it's wise to remove the #if as a while and just put the `Serializable` interface there. Agree?

**Comment by abrca** [comment]
Yes, good idea.

**Comment by GeertvanHorrik** [comment]
looks fantastic, thanks!

**Comment by GeertvanHorrik** [comment]
see https://marketplace.visualstudio.com/items?itemName=Catel.Catel-CodeSnippets

**Comment by abrca** [comment]
It might be better to split tag to two - 'catel' and 'snippets' ?

**Comment by GeertvanHorrik** [comment]
Excellent point, re-releasintg

**Pull request merged**